catroid/src/main/java/org/catrobat/catroid/stage/ShowBubbleActor.java

```diff
@@ -74,18 +77,52 @@ private void init() {
 	}
 
 	private Image getImageForDraw() {
-		if (drawRight) {
-			image.setX(sprite.look.getXInUserInterfaceDimensionUnit() + (sprite.look
-					.getWidthInUserInterfaceDimensionUnit() / 2));
+
+		if (sprite.look.getLookData() != null) {
```

Comment:
I would prefer the use of local variables to resolve calls to `something.somethingelse.somefunction()`